frontend/public/components/edit-yaml.jsx

```diff
@@ -86,8 +93,24 @@ export const EditYAML_ = connect(stateToProps)(
         this.setState({ error, success: null });
       }
 
+      loadCSVs() {
+        const { namespace } = this.props.obj.metadata;
```

Comment:
You should skip listing CSVs if there is no namespace.

This is problematic because the namespace can be edited in the YAML editor, and we won't get the updates here.

Reply:
The namespace can only change when creating a new resource. I think we'd want to skip listing CSVs for the creation editor. (If you're creating something, it's never owned by an operator.)